canonical-websites/www.ubuntu.com
# Build German version of VMWare to OpenStack takeover

## Done

- Built German VMWare to OpenStack takeover
- Created `index_de.html` for German users, which replaces the AI/ML takeover in the rotation with the new VMWare takeover (in German)
- Moved common markup between English and German index pages from `index.html` to `base_index.html`

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Check that the takeover alternates between the IoT App Store one and the AI/ML one
- Go to http://0.0.0.0:8001/index_de
- Check that the takeover alternates between the new VMWare to OpenStack one and the IoT App Store one
- Check that the copy matches the [brief](https://docs.google.com/document/d/1UkzAev5PKY5iR1mJOGNzpclxayATiImqE1AFO5wqXhI/edit?ts=5b7ecc0b)
- Check that it matches the original [design](https://github.com/canonical-websites/www.ubuntu.com/issues/3938)

## Issue / Card

Fixes #3939 

## Screenshots

![0 0 0 0_8001_ 2](https://user-images.githubusercontent.com/25733845/44844966-2a2ac600-ac44-11e8-91f5-bede20f3ae66.png)
